.packit.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+jobs:
+  # All tests specified in the `/plans/` subdir
+  # Podman e2e tests for Fedora and CentOS Stream
+  - job: tests
+    trigger: pull_request
+      #notifications:
+      #failure_comment:
+      #  message: "podman e2e tests failed. @containers/packit-build please check."
```

Comment:
might be good to have a separate github team listed for failure notification comments.